modules/imgcodecs/test/test_exr.impl.hpp

```diff
@@ -23,6 +34,7 @@ TEST(Imgcodecs_EXR, readWrite_32FC1)
     ASSERT_EQ(CV_32FC1,img.type());
 
     ASSERT_TRUE(cv::imwrite(filenameOutput, img));
+    ASSERT_EQ(getFileSize(filenameOutput), 396);
```

Comment:
Please add comment why you check file size here. It's important for test, but not obvious for other contributors.